src/Tizen.Network.NetworkMonitor/Tizen.Network.NetworkMonitor/Connection.cs

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using Tizen.Network;
+
+namespace Tizen.Network.NetworkMonitor
+{
+    /// <summary>
+    /// A class which manages connection information.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class Connection : IDisposable
```

Comment:
What is different from `Tizen.Network.Connection.ConnectionItem`

Reply:
This is similar to Tizen.Network.ConnectionProfile rather than ConnectionItem.
It is included only for the purpose of obtaining network connection information. In addition, it has GetLink() method to obtain network interface information.